### PR TITLE
add the auto-generated tabular form for examples page

### PR DESCRIPTION
**Pull request by AemieJ**
* As the example page doesn't include the list of examples in the tabular form, I have made changes to the examples page to include the autogenerated tabular form.

* However, the link to each example is to the `README.adoc` within GitHub itself, as there is no separate page for each example. Also, there doesn't exist a `README.adoc` for **timer-log-kotlin**, it direct to the GitHub repo for it.

### CODE REVIEW

File: docs/modules/ROOT/pages/examples.adoc [hunk 1]
```diff
@@ -2,3 +2,19 @@
...
+| link:https://github.com/apache/camel-quarkus/tree/master/examples/timer-log/README.adoc[Timer Log] (timer-log) | An example of basic Hello World that uses a Camel timer
+| link:https://github.com/apache/camel-quarkus/tree/master/examples/timer-log-cdi/README.adoc[Timer Log CDI] (timer-log-cdi) | An example of basic Hello World that uses CDI to set-up a Camel timer
+| link:https://github.com/apache/camel-quarkus/tree/master/examples/timer-log-kotlin[Timer Log Kotlin] (timer-log-kotlin) | An example of basic Hello World that uses Kotlin to set-up a Camel timer
+| link:https://github.com/apache/camel-quarkus/tree/master/examples/timer-log-xml/README.adoc[Timer Log XML] (timer-log-xml) | An example of basic Hello World that uses XML to set-up a Camel timer
+| link:https://github.com/apache/camel-quarkus/tree/master/examples/timer-log-spring/README.adoc[Timer Log Spring] (timer-log-spring) | An example of basic Hello World that uses Spring to set-up a Camel timer
+| link:https://github.com/apache/camel-quarkus/tree/master/examplesfile-split-log-xml/README.adoc[File Split Log XML] (file-split-log-xml) | An example of basic Hello World that uses XML to set-up a Camel file consumer
```
Comment (jamesnetherton) [inline docs/modules/ROOT/pages/examples.adoc]: I think you're missing a '/' in the URL here

Comment (AemieJ) [reply docs/modules/ROOT/pages/examples.adoc]: Which URL? Also for **timer-log-kotlin** there doesn't exist any **README.adoc**

Comment (jamesnetherton) [reply docs/modules/ROOT/pages/examples.adoc]: For file-split-log-xml. It has `examplesfile-split-log-xml`.

Comment (AemieJ) [reply docs/modules/ROOT/pages/examples.adoc]: Thank you, I will fix that.

Comment (AemieJ) [reply docs/modules/ROOT/pages/examples.adoc]: Fixed that problem with the URL.

**Comment by ppalaga** [comment]
@AemieJ the commit message says auto-generated, but where is the code that generates the table in the commit?

**Comment by AemieJ** [comment]
> @AemieJ the commit message says auto-generated, but where is the code that generates the table in the commit?

Isn't it the syntax of AsicDoctor that you write the table data in particular syntax and it will provide you with a responsive table design?

**Comment by ppalaga** [comment]
> > @AemieJ the commit message says auto-generated, but where is the code that generates the table in the commit?
> 
> Isn't it the syntax of AsicDoctor that you write the table data in particular syntax and it will provide you with a responsive table design?

Sorry, there is probably some mismatch between you and me in how we understand the term "auto-generated". For me, auto-generated text is an output of some computer program. The list of extensions https://camel.apache.org/camel-quarkus/latest/list-of-camel-quarkus-extensions.html is auto-generated in this sense by the maven plugin located in our repo: https://github.com/apache/camel-quarkus/blob/master/tooling/package-maven-plugin/src/main/java/org/apache/camel/quarkus/maven/UpdateDocExtensionsListMojo.java 
This approach has the advantage that anytime a new extension is added the list is updated automatically. 

I thought, you were doing something similar for the list of examples. If this was not your aim, I am fine with that, the change is an improvement against the previous state. We do not add new examples that often, so the list can be updated manually. But please remove the word `auto-generated` from the commit message and ideally squash your commits.

**Comment by AemieJ** [comment]
> > > @AemieJ the commit message says auto-generated, but where is the code that generates the table in the commit?
> > 
> > 
> > Isn't it the syntax of AsicDoctor that you write the table data in particular syntax and it will provide you with a responsive table design?
> 
> Sorry, there is probably some mismatch between you and me in how we understand the term "auto-generated". For me, auto-generated text is an output of some computer program. The list of extensions https://camel.apache.org/camel-quarkus/latest/list-of-camel-quarkus-extensions.html is auto-generated in this sense by the maven plugin located in our repo: https://github.com/apache/camel-quarkus/blob/master/tooling/package-maven-plugin/src/main/java/org/apache/camel/quarkus/maven/UpdateDocExtensionsListMojo.java
> This approach has the advantage that anytime a new extension is added the list is updated automatically.
> 
> I thought, you were doing something similar for the list of examples. If this was not your aim, I am fine with that, the change is an improvement against the previous state. We do not add new examples that often, so the list can be updated manually. But please remove the word `auto-generated` from the commit message and ideally squash your commits.

Alright, I have currently made a new PR with a different commit.

**Comment by ppalaga** [comment]
BTW you could do it in the current pull request by squashing them frist https://thoughtbot.com/blog/git-interactive-rebase-squash-amend-rewriting-history and then force pushing to the `examples` in your fork https://stackoverflow.com/questions/10510462/force-git-push-to-overwrite-remote-files That would change the commits in this pull request.

**Pull request closed without merging**